src/components/Images/Images.scss

```diff
@@ -3,14 +3,14 @@
 
 // WIP.
 
+/* Generally used as a wrapper around Image components. */
 figure {
   margin: 0;
 }
 
 .Image {
```

Comment:
It would be nice if this stuff could be true CSS Modules and not a global

Reply:
converted